validator/sawtooth_validator/journal/consensus/poet/poet_consensus/poet_block_publisher.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import logging
+import hashlib
+import time
+import json
+
+from sawtooth_validator.journal.consensus.consensus \
+    import BlockPublisherInterface
+
+from sawtooth_validator.journal.consensus.poet.poet_consensus \
+    import poet_enclave_factory as factory
+from sawtooth_validator.journal.consensus.poet.signup_info import SignupInfo
+from sawtooth_validator.journal.consensus.poet.wait_timer import WaitTimer
+from sawtooth_validator.journal.consensus.poet.wait_certificate \
+    import WaitCertificate
+from sawtooth_validator.journal.consensus.poet.poet_consensus import utils
+
+from sawtooth_validator.journal.block_wrapper import NULL_BLOCK_IDENTIFIER
+
+import sawtooth_validator.protobuf.transaction_pb2 as txn_pb
+import sawtooth_validator.protobuf.validator_registry_pb2 as vr_pb
+
+from sawtooth_signing import secp256k1_signer as signing
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PoetBlockPublisher(BlockPublisherInterface):
+    """Consensus objects provide the following services to the Journal:
+    1) Build candidate blocks ( this temporary until the block types are
+    combined into a single block type)
+    2) Check if it is time to claim the current candidate blocks.
+    3) Provide the data a signatures required for a block to be validated by
+    other consensus algorithms
+    """
+
+    # HACER: For now, we need to keep some sealed signup data and serialized
```

Comment:
=)  You should have seen how I used to get by it in the past:

# Try to create a sentence so it is not
# Obvious that the comment was
# Designed so that first letters from
# Opening word to end spelled TODO